test/image/mocks/multicategory_series.json

```diff
@@ -0,0 +1,1617 @@
+                "dash": "dot",
+                "shape": "spline",
+                "color": "#737373"
+            },
+            "hovertemplate": "",
+            "x": [
+                [
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low"
+                ],
+                [
+                    0,
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    6,
+                    7,
+                    8,
+                    9,
+                    10,
+                    11
+                ]
+            ],
+            "y": [
+                295,
+                300,
+                275,
+                240,
+                195,
+                155,
+                120,
+                100,
+                80,
+                75,
+                90,
+                115
+            ],
+            "name": "B - 4",
+            "yaxis": "y",
+            "legendgroup": "B - 4",
+            "hoverinfo": "none",
+            "showlegend": false
+        },
+        {
+            "type": "scatter",
+            "mode": "lines",
+            "line": {
+                "dash": "dot",
+                "shape": "spline",
+                "color": "#737373"
+            },
+            "hovertemplate": "",
+            "x": [
+                [
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low"
+                ],
+                [
+                    0,
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    6,
+                    7,
+                    8,
+                    9,
+                    10,
+                    11
+                ]
+            ],
+            "y": [
+                295,
+                300,
+                275,
+                240,
+                195,
+                155,
+                120,
+                100,
+                80,
+                75,
+                90,
+                115
+            ],
+            "name": "A - 4",
+            "yaxis": "y",
+            "legendgroup": "A - 4",
+            "hoverinfo": "none",
+            "showlegend": false
+        }
+    ],
+    "layout": {
+        "autosize": true,
+        "margin": {
+            "t": 40
+        },
+        "legend": {
+            "x": 0.5,
+            "y": "auto",
```

Comment:
Let's remove this line so that the `npm run test-mock multicategory_series" would pass.
```
"In layout, key legend.y is set to an invalid value (auto)"
```